docs/tutorials/projects/make-a-plant-watering-robot.md

```diff
@@ -268,7 +279,7 @@ First, add your Pi as a [board component](/components/board/) by creating a new
 
 Then, add your pump as a [motor component](/components/motor/) by adding a new component with **type** `motor` and **model** `gpio`.
 
-Add your board (in this example, named `local`) to this component's dependencies (`depends_on`) and set the attributes **Max RPM** to `1000` and **PWM**  to `12` (the pin # that you wired the PWM to).
+Add your board (in this example, named `local`) to this component's dependencies (`depends_on`) and set the attributes **Max RPM** to `1000` and **PWM**  to `12 GPIO 18` (the number and GPIO number of the pin that you wired the pump's PWM to).
```

Comment:
```suggestion
Select your board (in this example, named `local`) from the **Board** drop-down and set the attributes **Max RPM** to `1000` and **PWM**  to `12 GPIO 18` (the number and GPIO number of the pin that you wired the pump's PWM to).
```
You do not need to put it in the depends_on because that's taken care of implicitly when you add it as the board. I recommend taking it out of the example JSON as well. If you choose to leave it in, it should at least be indented in the JSON.